src/main/java/org/fundacionjala/coding/maria/Kata5.java

```diff
@@ -0,0 +1,29 @@
+package org.fundacionjala.coding.maria;
+
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Kata 5.
+ * Complementary DNA
+ */
+public class Kata5 {
+    /**
+     * @param dna DNA
+     * @return complement to DNA
+     */
+    public String makeComplement(String dna) { //ATTGC   -> TAACG
+        Map<String, String> comp = new HashMap<String, String>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)